### PR TITLE
[Work In Progress] Attempt to fix #133

### LINKED ISSUE #133: Nav-bar item activate doesn't work in Dropdown
### What steps will reproduce the problem?

Open the class yii\bootstrap4\Dropdown

### What's expected?

Using $itemOptions for activate link/item in Nav-bar (and for over options).

### What do you get instead?

$itemOptions not used and in result - we can't activate menu-item.

### PR DESCRIPTION
**Pull request by Thoulah**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #133

Needs further testing before merge

### CODE REVIEW

File: tests/NavTest.php [hunk 1]
```diff
@@ -196,8 +196,8 @@ public function testImplicitActive()
         ]);
 
         $expected = <<<EXPECTED
-<ul id="w0" class="nav"><li class="nav-item active"><a class="nav-link active" href="#">Item1</a></li>
-<li class="nav-item active"><a class="nav-link active" href="/base/index.php?r=site%2Findex">Item2</a></li></ul>
```
Comment (Thoulah) [inline tests/NavTest.php]: Complies with https://getbootstrap.com/docs/4.3/components/navs/#base-nav

**Comment by samdark** [comment]
Merged. Thank you!

**Comment by samdark** [comment]
Reverted it.

**Comment by Thoulah** [comment]
Hi @samdark 

Do you know the best way to update my fork from master and apply these changes again for PR?

**Comment by samdark** [comment]
Updating is easy: https://github.com/yiisoft/yii2/blob/master/docs/internals/git-workflow.md#7-pull-the-latest-yii-code-from-upstream-into-your-branch

To apply changes one more time something like [`hub am`](https://hub.github.com/) can solve it.

**Pull request merged**